.eslintrc

```diff
@@ -0,0 +1,206 @@
+{
+  "env": {
+    "browser": true,
+    "node": true,
+    "es6": true
+  },
+  "parserOptions": {
+    "sourceType": "module"
+  },
+  "globals": {
+    "estatico": true,
+    "Modernizr": false,
+    "open": true,
+    "name": true,
+    "Promise": false
+  },
+  "rules": {
+    "no-bitwise": 2,
```

Comment:
According to the documentation http://eslint.org/docs/rules/no-bitwise and even to previous versions of the documentation (https://github.com/eslint/eslint/blob/master/docs/rules/no-bitwise.md) `"no-bitwise"` should not admit a number as a value. Additionally, configuring `"extends": "eslint:recommended"` would also bring this by default as an error, and if ESLint changes the API for this rule we wouldn't need to worry about it in the future.